index.html

```diff
@@ -711,6 +711,15 @@ <h3 id="miscellaneous"> Longer Miscellaneous </h3>
     <td><img src='https://img.shields.io/ansible/role/3078.svg' alt=''/></td>
     <td><code>https://img.shields.io/ansible/role/3078.svg</code></td>
   </tr>
+  <tr><th> Libraries.io for releases: </th>
+    <td><img src='https://img.shields.io/librariesio/release/hex/phoenix/1.0.3.svg' alt=''/></td>
+    <td><code>https://img.shields.io/librariesio/release/hex/phoenix/1.0.3.svg</code></td>
+  <tr><th> Libraries.io for GitHub: </th>
+    <td><img src='https://img.shields.io/librariesio/github/phoenixframework/phoenix.svg' alt=''/></td>
+    <td><code>https://img.shields.io/librariesio/github/phoenixframework/phoenix.svg</code></td>
+
+  </tr>
+
```

Comment:
index.html is generated, so this change won't survive a `make website`. I believe you want to put it in try.html.